Eatery/Controllers/MealTableViewController.swift

```diff
@@ -16,11 +15,18 @@ class MealTableViewController: UITableViewController {
         }
     }
 
+    private let dateFormatter = DateFormatter()
```

Comment:
Should this be duplicated in EateriesViewController? Maybe find some way to abstract this out if the format ever changes.